dataplane/src/main.rs

```diff
@@ -1,6 +1,9 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright Open Network Fabric Authors
 
+mod config;
+mod pipeline;
```

Comment:
Nit: Some other modules are defined below, _after_ the list of `use ...`. I don't know what's the established practice, but whether it's before or after the `use ...` block, can we have them all at a single location please?

Reply:
Fixed to put them all above use definitions, as per @daniel-noland's suggestion.